test/fixtures/poro.rb

```diff
@@ -33,6 +38,10 @@ def id
     @attributes[:id] || @attributes['id'] || object_id
   end
 
+  def to_param
+    id.to_s
+  end
```

Comment:
:+1:  Nice, could you add some comments on top of those method explaining this to help new contributors when jumping into the code :smile: